crates/uv-cli/src/lib.rs

```diff
@@ -2697,9 +2697,10 @@ pub struct RunArgs {
     #[arg(long, overrides_with("no_dev"), hide = true)]
     pub dev: bool,
 
-    /// Omit the development dependency group.
+    /// Disable the development dependency group.
     ///
     /// This option is an alias of `--no-group dev`.
+    /// You probably actually want `--no-default-groups`.
```

Comment:
This is a bit more casual than we usually use. I'd say

> See `--no-default-groups` to disable all default groups instead.

I'm a little hesitant to have this here, but it seems okay. Generally, `--no-dev` and `--dev` exist for users that don't care about using multiple groups.